src/components/Dialogs/Dialogs.js

```diff
@@ -1,25 +1,19 @@
 import React from 'react';
-import { NavLink } from 'react-router-dom';
 
-import { dialogs, messages } from '../../index';
+import DialogItem from './DialogsItem/DialogsItem';
+import Message from './Message/Message';
 
 import style from './Dialogs.module.css';
 
-const DialogItem = (props) => {
-  const path = `/dialogs/${props.id}`;
+const Dialogs = (props) => {
+  const dialogsElements = props.state.dialogs.map((d) => <DialogItem name={d.name} id={d.id} />);
```

Comment:
Consider adding a new const for "props.state.dialogs".

Reply:
/fxd